repose-aggregator/src/docs/asciidoc/developer/docker.adoc

```diff
@@ -0,0 +1,47 @@
+= Repose and Docker - Tips and Tricks
+
+This is a collection of information cobbled together from several sources, but mostly from sifting through the https://docs.docker.com/[Official Docker Documentation] and then applying it to *Repose*.
+
+The rest of this assumes you are already familiar with the contents of the <<../recipes/quick-start.adoc#,Quick Start>> and <<../recipes/docker.adoc#,Docker>> recipes.
+Obviously since you made it here, your hunger for *Repose Docker* knowledge was not satiated.
+
+== Overview
```

Comment:
Overview of what? The steps listed are specifically for running Repose in Docker, so I'm not a fan of just calling this section overview.

Reply:
Updated.